tests/tests.yaml

```diff
@@ -1,13 +1,24 @@
 charm_name: none
 gate_bundles:
-- magpie-xenial
-- magpie-bionic
-- magpie-focal
-dev_bundles:
-# magpie isn't hirsute ready yet.
-- magpie-hirsute
-target_deploy_status:
-  magpie:
+# let's keep this list as short as possible, this is not a gate test for any of
+# the charms in the bundle but a test for Zaza's handling of model deployment/
+# destruction
+- first
+- second
+arget_deploy_status:
```

Comment:
s/arget/target/

Reply:
Thank you for pointing that out! I was wondering what caused it to not pass this time :)